packages/grafana-data/src/valueFormats/dateTimeFormatters.ts

```diff
@@ -102,6 +102,9 @@ export function toSeconds(size: number, decimals?: DecimalCount, scaledDecimals?
   if (size === null) {
     return { text: '' };
   }
+  if (size === 0) {
+    return { text: '0 s' };
```

Comment:
```suggestion
    return { text: '0', suffix:' s', numeric:0 };
```

Maybe

Reply:
`FormattedValue` has no such member "numeric" but sure if you'd like the suffix separated then why not